packages/sheets/src/facade/f-workbook.ts

```diff
@@ -744,10 +745,61 @@ export class FWorkbook extends FBaseInitialable {
      * const builder = activeSpreadsheet.getDefinedName('MyDefinedName').toBuilder();
      * builder.setRef('Sheet1!A2').setName('MyDefinedName1').build();
      * activeSpreadsheet.updateDefinedNameBuilder(param);
-     *
      * ```
      */
     updateDefinedNameBuilder(param: ISetDefinedNameMutationParam): void {
         this._commandService.syncExecuteCommand(SetDefinedNameCommand.id, param);
     }
+
+    /**
+     * Gets the registered range themes.
+     * @returns {string[]} The name list of registered range themes.
+     * @example
+     * ```ts
+     * // The code below gets the registered range themes
+     * const activeSpreadsheet = univerAPI.getActiveWorkbook();
+     * const themes = activeSpreadsheet.getRegisteredRangeThemes();
+     * console.log(themes);
+     * ```
+     */
+    getRegisteredRangeThemes(): string[] {
+        return this._injector.get(SheetRangeThemeService).getRegisteredRangeThemes();
+    }
+
+    /**
+     * Register a custom range theme style.
+     * @example
+     * ```ts
+     * // import {RangeThemeStyle} from '@univerjs/sheets';
+     * const fWorkbook = univerAPI.getActiveWorkbook();
+     * const rangeThemeStyle = new RangeThemeStyle('MyTheme');
+     * rangeThemeStyle.setSecondRowStyle({
+     *    bg: {
+     *       rgb: 'rgb(214,231,241)',
+     *    },
+     * });
+     * fWorkbook.registerRangeTheme(rangeThemeStyle);
+     * ```
+     */
+    registerRangeTheme(rangeThemeStyle: RangeThemeStyle): void {
+        this._commandService.syncExecuteCommand(RegisterWorksheetRangeThemeStyleCommand.id, {
+            unitId: this.getId(),
+            rangeThemeStyle,
+        });
+    }
+
+    /**
+     * Unregister a custom range theme style.
+     * @example
+     * ```ts
+     * const fWorkbook = univerAPI.getActiveWorkbook();
+     * fWorkbook.unRegisterRangeTheme('MyTheme');
+     * ```
+     */
+    unRegisterRangeTheme(themeName: string): void {
```

Comment:
To make it consistent with the command's name:

```suggestion
    unregisterRangeTheme(themeName: string): void {
```